### PR TITLE
Docs(output) : Explictily Add info about deafult values

### PR DESCRIPTION
**Pull request by jeffin143**
_describe your changes..._

In view of 
https://github.com/webpack/webpack.js.org/issues/3313#issuecomment-587923993 
#3313 
- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

File: src/content/configuration/output.md [hunk 1]
```diff
@@ -1072,6 +1072,7 @@ module.exports = {
   }
 };
 ```
+T> Deafult ecmaVersion is 2015(6) for webpack5 and for webpack4 it is 5.
```
Comment (jamesgeorge007) [inline src/content/configuration/output.md]:
```suggestion
T> Default ecmaVersion is 2015(6) for webpack5 and for webpack4 it is 5.
```

Comment (EugeneHlushko) [inline src/content/configuration/output.md]: When we reference specific webpack version, the format is following:
- webpack 4
- webpack 5

It is also mentioned in [contribution guide](https://webpack.js.org/contribute/writers-guide/#typesetting)

We mention types and default after option title, remove the info about webpack 5 as the docs are for webpack 5 explicitly and only keep the warning mentioning default for webpack 4

```suggestion
T> The dafault value of `output.ecmaVersion` in webpack 4 is `5`.
```

**Comment by EugeneHlushko** [comment]
@jeffin143 i've added type and default here: #3591 so you dont have to worry about, just update the warning

**Comment by EugeneHlushko** [comment]
```
$ markdownlint --rules markdownlint-rule-emphasis-style --config ./.markdownlint.json *.md ./src/content/**/*.md --ignore './src/content/**/_*.md'
./src/content/configuration/output.md: 1119: MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
error Command failed with exit code 1.
```

File: src/content/configuration/output.md [hunk 2]
```diff
@@ -1072,6 +1072,7 @@ module.exports = {
   }
 };
 ```
+T> The dafault value of `output.ecmaVersion` in webpack 4 is `5`.
```
Comment (EugeneHlushko) [inline src/content/configuration/output.md]:
```suggestion

T> The dafault value of `output.ecmaVersion` in webpack 4 is `5`.
```

**Comment by EugeneHlushko** [comment]
lets see if it passes now, should b good

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**